src/pages/post-feed/postListItem.tsx

```diff
@@ -0,0 +1,112 @@
+import Link from 'next/link';
+import Image from 'next/image';
+import { useTranslation } from 'next-i18next';
+import { FacebookIcon, LinkedInIcon, TwitterIcon, EmailIcon, SandwichButton } from './icons';
+import { User } from './usersDummyData';
+
+type Props = {
+  user: User;
+};
```

Comment:
postListItem should receive a post as props, or even better only a few props of the post that it needs to render

```suggestion
type Props = {
  post: Post;
};
```

Reply:
and with the post data you could have a nested obj for the user data

something like this
```ts
type Post = {
  title: string;
  body: string;
  image: string // the image link for the post and not the user
  creator: {
    fullName: string;
    id: string;
    avatar: string;
  }
}
```

you could in the future have the likes and the comments counts as well